src/backend/aspen/app/views/phylo_trees.py

```diff
@@ -163,6 +140,37 @@ def sample_filter(sample: Sample):
     )
     json_data = json.loads(data)
 
+    if id_style == "public":
+        return json_data
```

Comment:
If the user only wanted public ID's, we can just return the raw tree data and skip the rest of the code in this function that converts ID's.